pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/V1Constants.java

```diff
@@ -65,6 +65,8 @@ public static class MetadataKeys {
     public static class Segment {
       public static final String SEGMENT_CREATOR_VERSION = "creator.version";
       public static final String SEGMENT_NAME = "segment.name";
+
+      public static final String SEGMENT_METADATA_VERSION = "segment.metadata.version";
```

Comment:
Is this used? I cannot find any usage of it

Reply:
I am planning to use this constant for writing new segment metadata, however that work I am planning to do in two other PRs. So, this is unused from this PR perspective.